dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/client/migration/MigrationInvoker.java

```diff
@@ -449,10 +452,10 @@ protected void refreshInterfaceInvoker(CountDownLatch latch) {
         }
         setListener(invoker, () -> {
             latch.countDown();
-            //TODO FrameworkStatusReporter
-//            FrameworkStatusReporter.reportConsumptionStatus(
-//                createConsumptionReport(consumerUrl.getServiceInterface(), consumerUrl.getVersion(), consumerUrl.getGroup(), "interface")
-//            );
+            if (reportService.hasReporter()) {
+                reportService.reportMigrationStepStatus(
```

Comment:
change to `reportService.reportConsumptionStatus`

Reply:
done